OpenRA.Mods.Common/Scripting/Properties/HelicopterProperties.cs

```diff
@@ -16,7 +16,7 @@
 namespace OpenRA.Mods.Common.Scripting
 {
 	[ScriptPropertyGroup("Movement")]
-	public class HelicopterProperties : ScriptActorProperties, Requires<HelicopterInfo>
+	public class HelicopterProperties : ScriptActorProperties, Requires<AircraftInfo>
```

Comment:
The `Move` property is going to conflict with the one in `PlaneProperties`.  Could you please rename `PlaneProperties` &rarr; `AircraftProperties` (won't affect the lua compatibility, as that isn't exposed), and then add the heli/plane checks there?